src/lib/free-queue/README.md

```diff
@@ -1 +1,114 @@
-### Work In Progress ....
\ No newline at end of file
+# Audio Buffering Library
+
+## Introduction
+
+This library provides two classes, `FreeQueue` and `FreeQueueSAB`, designed to handle audio buffering using WebAssembly (WASM) and SharedArrayBuffer (SAB) respectively. These classes offer efficient ways to manage audio data, making them suitable for applications requiring low-latency audio processing.
```

Comment:
The benefit of a ring buffer in the audio application is to bring the flexibility and the smooth integration with other parts. A good ring buffer implementation also ensure the minimum latency possible - but it adds more latency than non-buffering solution.